process_report/process_report.py

```diff
@@ -217,9 +228,10 @@ def main():
             invoice_list.append(os.path.join(args.output_folder, pi_invoice))
 
         upload_to_s3(invoice_list, invoice_month)
+        upload_to_s3_old_pi_file(old_pi_file)
 
 
-def fetch_S3_invoices(invoice_month):
+def fetch_s3_invoices(invoice_month):
```

Comment:
While yes, this should have been `s3` instead of `S3` and in python things are preferred to be written in lower case snake case. Please try to refrain from introducing unrelated changes to a PR. Do not revert it for now, this is fine.